lib/index.js

```diff
@@ -11,6 +11,7 @@ var crypto = require('crypto');
 var database = require('bedrock-mongodb');
 var util = require('util');
 var BedrockError = bedrock.util.BedrockError;
+require('bedrock-server');
```

Comment:
Why was `bedrock-server` added? Requiring this makes it so that this module can't be used in applications that don't have a server ... so it would be good if we can figure out a way to avoid introducing this dependency. If it was already used somewhere and this is a "fix", perhaps we can eliminate its usage somehow.